pkgs/development/php-packages/relay/default.nix

```diff
@@ -14,42 +15,36 @@
 }:
 
 let
-  version = "0.7.0";
+  version = "0.10.0";
   hashes = {
     "aarch64-darwin" = {
       platform = "darwin-arm64";
       hash = {
-        "8.0" = "sha256-pd/9TWZPgAfmVM0/QVYRHu5k4gANcxCSnfAl38irO0Y=";
-        "8.1" = "sha256-OpxE/nu8MZedTmKGQeyJm36pyyHlRpW11avuGcnGP68=";
-        "8.2" = "sha256-+CMPdXZotUr43Qda1FwGpuWPEE1K4RuBNE9fiokAtoY=";
-        "8.3" = "sha256-lbKVxOd5gK5VDGnJ42w7L5DFKsBQDZXgEZLR/Y0gP88=";
+        "8.1" = "sha256-aEuYKo31dKV7TSOeKt4BSShstNxfS4EdibJ2279XTbg=";
+        "8.2" = "sha256-tjE+bAiVWYh6od8rW7flZ6ajMGxMJszw7H055VtDJsc=";
+        "8.3" = "sha256-p5GjuMe76Nb/KMzF8kLOa3rN538udCQhMmAEFp0u8bM=";
+        "8.4" = "sha256-QUryARS5omADR3kEykCnoK4IFau1RpTQKDcCJ+lN/SY=";
+        "8.5" = "cvpPmXm9rXjs1QmHbYv5aEbWnsb/t1pCAzutTrPDY0w=";
```

Comment:
Why some hashes are not in the same format as the others?

Reply:
Some of the hashes I fetched manually through `nix-prefetch-url`. Not sure if it's the best way

Reply:
But 8.5 is not yet available in nixpkgs, I don't think it's relevant to add them now. Don't you think ?

Reply:
It's relevant in a way of not having to update this file once PHP 8.5 is available in nixpkgs. It will already be working with PHP 8.5 if we include these php 8.5 hashes already now.